src/Rules/Validation/ScopeIlluminateRequestValidateMethods.php

```diff
@@ -0,0 +1,58 @@
+<?php declare(strict_types=1);
+
+namespace Hihaho\PhpstanRules\Rules\Validation;
+
+use Illuminate\Http\Request as IlluminateRequest;
+use PhpParser\Node;
+use PhpParser\Node\Expr\MethodCall;
+use PHPStan\Analyser\Scope;
+use PHPStan\Rules\RuleErrorBuilder;
+use PHPStan\ShouldNotHappenException;
+use ReflectionException;
+
+final class ScopeIlluminateRequestValidateMethods extends ScopeValidationMethods
+{
+    public function getNodeType(): string
+    {
+        return MethodCall::class;
+    }
+
+    /**
+     * @phpstan-param MethodCall $node
+     * @throws ShouldNotHappenException | ReflectionException
+     */
+    public function processNode(Node $node, Scope $scope): array
+    {
+        if (str_starts_with($scope->getNamespace(), 'App\\Http\\Request')) {
+            return [];
+        }
+
+        if (! $this->hasIlluminateRequestClass($scope)) {
+            return [];
+        }
+
+        if (! $this->isValidateMethod($this->nameFrom($node))) {
+            return [];
+        }
+
+        return [
+            RuleErrorBuilder::message(
+                'Usage of unvalidated request data is not allowed outside of App\\Http\\Requests'
+            )
+                ->nonIgnorable()
+                ->tip('Use $request->safe() to use request data')
+                ->identifier('hihaho.request.unsafeRequestData')
+                ->build(),
+        ];
+    }
+
+    /** @throws ReflectionException */
+    private function hasIlluminateRequestClass(Scope $scope): bool
```

Comment:
this check is a bit simpler than the FormRequest one. Reason is that we directly have a class we need to check. So no need for extra checks or conversions.